src/main/java/nl/finnt730/PasteCommand.java

```diff
@@ -126,10 +123,12 @@ private void createPaste(String content, net.dv8tion.jda.api.entities.channel.mi
                      String pasteUrl = extractUrlFromResponse(responseStr);
                      String rawUrl = extractRawUrlFromResponse(responseStr);
                      if (pasteUrl != null && rawUrl != null) {
-                         channel.sendMessage("📓 **Paste created for " + user.getAsMention() + "!**\n" +
-                                 "**Paste URL:** " + pasteUrl + "\n" +
-                                 "**Raw URL:** " + rawUrl + "\n" +
-                                 "*This paste will be automatically deleted 90 days after the last view.*").queue();
+                         String sentContent = String.format(
+                                 "**Paste created by %s**! \n" +
+                                 "[PASTE URL](<%s>) | [RAW URL](<%s>) \n" +
+                                 "This paste will be automatically deleted 90 days after the last view."
```

Comment:
Please make this just:
`[Paste](<%s>) | [Raw](<%s>)`
As a 1 line message is enough.